server/controllers/stock/core.js

```diff
@@ -16,6 +16,7 @@ const moment = require('moment');
 const db = require('../../lib/db');
 const FilterParser = require('../../lib/filter');
 const util = require('../../lib/util');
+// const { param } = require('jquery');  ??? Not sure why hound is complaining about this now
```

Comment:
You don't have to insert this line after local require `require('../../a/given/path')`

If you put it after `const moment = require('moment');` there will be no complain I think

Reply:
This did not help.  I'm leaving it commented out.  I suppose we should track down why it is no longer needed or delete the commented out version.